p2pnetwork/node.py

```diff
@@ -33,7 +33,7 @@ def node_callback(event, main_node, connected_node, data):
                  connected_node: Which connected node caused the event.
                  data: The data that is send by the connected node."""
 
-    def __init__(self, host, port, callback=None):
+    def __init__(self, host, port, callback=None, max_connections=0):
```

Comment:
Is this the correct place it on this position in de constructor? To use the max_connections, you also need to use the callback. I would like to propose an option variabele or named variables?

Reply:
Yes, is the correct place to put named arguments (keyword arguments). You don't need to use callback function to set max connections. When overwrite the method we just use super() to setup in the the parent class constructor arguments.